src/app/components/authorization/authorization.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useState, useRef } from 'react'
```

Comment:
All react components have to be named from capital letter. Rename `authorization.tsx` to `Authorization.tsx`